### PR TITLE
REL- 2571: Add help page links to PIT tabs

### PR DESCRIPTION
**Pull request by cquiroz**
Small PR that adds a tooltip to the help button on the PIT sub windows

### CODE REVIEW

File: bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java [hunk 1]
```diff
@@ -320,7 +274,7 @@ public void setSelected(boolean newValue) {
      * @param bar     the panel's tool bar
      * @return the panel's built header area
      */
-    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon) {
+    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon, String hint) {
```
Comment (sraaphorst) [inline bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java]: Apart from `null` distaste and wondering if we should instead use an `Option<String>` here, with a `hintOption.foreach(help::setToolTipText)`,  :+1:

Comment (cquiroz) [inline bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java]: Should we keep using the old Java `Option` class? Or do we prefer `Optional` for Java 1.8

Comment (sraaphorst) [inline bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java]: That's something I've also been wondering about. In most new code I write, I use Java 8 `Optional`, and in old code, I keep Gemini `Option`, and nobody seems to have complained thus far :smile:

**Comment by cquiroz** [comment]
Went using `Optional` and removed the use of nulls. A small case class encapsulates the parameters for the help button

**Pull request merged**